doc/Makefile.am

```diff
@@ -1,8 +0,0 @@
-# Makefile.am for testsuite/doc
-
-SUBDIRS = autodocs
-
-htmldir = @docdir@
-html_DATA = $(wildcard *.html)
```

Comment:
Um, there is some non-auto generated doc, it might be outdated (the last change was ~14 years ago), but it should be kept IMHO.

Just removing the `SUBDIRS = autodocs` and removing the `include $(top_srcdir)/autodocs-ycp.ami` should be enough in this case.